stdlib/Test/src/Test.jl

```diff
@@ -1435,6 +1436,7 @@ function testset_forloop(args, testloop, source)
             err isa InterruptException && rethrow()
             # Something in the test block threw an error. Count that as an
             # error in this test set
+            ccall(:jl_test_failure_breakpoint, Cvoid, (Any,), err)
```

Comment:
Inserting a ccall in a macro will force a large amount of additional compilation and corrupt our test heuristics here. Can we outline this?